compiler/ballerina-parser/src/test/resources/expressions/query-expr/query_expr_source_126.bal

```diff
@@ -0,0 +1,5 @@
+function f1() {
+    _ = from var {x} in [{"x":2, "y":3}, {"x":4, "y":5}]
+            collect x
+            collect x;
```

Comment:
We need to test making `group` not being a keyword. Better not mix it with other parser tests.

Let's add a new class say, `ContextualKeywordTest.java` under the `misc` folder.
There, let's add two tests testing `collect` and `group` as identifiers. 
e.g.
```
int collect = 23;
int group = 4;
```